frontend/src/components/Humanboard.vue

```diff
@@ -91,37 +48,21 @@ export default {
   },
   data() {
     return {
-      dialog: false,
       radios: "white",
       opponentMoveFrom: "e7",
       opponentMoveTo: "e5",
-      promotion: "q",
       orientation: "white",
       pieceColor: "white",
       gameInfo: [],
       opponent: ""
     };
   },
   methods: {
-    sendMoveToOpponent(move) {
-      let color = this.game.turn() === "w" ? "white" : "black";
-      if (
-        this.gameInfo.players.player1Name === this.$store.state.loginUser &&
-        this.gameInfo.players.player1Color !== color
-      ) {
-        this.opponentMoveFrom = move.from;
-        this.opponentMoveTo = move.to;
-        this.promotion = move.promotion;
-        this.opponentMove();
-      } else if (
-        this.gameInfo.players.player2Name === this.$store.state.loginUser &&
-        this.gameInfo.players.player2Color !== color
-      ) {
-        this.opponentMoveFrom = move.from;
-        this.opponentMoveTo = move.to;
-        this.promotion = move.promotion;
-        this.opponentMove();
-      }
```

Comment:
Why you removed this func, is game works without it? It's a bad idea. Here we checking - send move or not

Reply:
Yes, the game works. I sent the data.move directly to this.opponentMove(data.move);
I have been checking the functionality under different circumstances and don't see any drawbacks.